docs/2_step_by_step/index.md

```diff
@@ -1,20 +0,0 @@
-# Step by step
```

Comment:
I see the dropdown is back, is that intentional?
![image](https://user-images.githubusercontent.com/44071807/174602749-bc8c1da0-5242-4f2a-8bc0-fb81a96b840c.png)
(but I think netlify is showing me an old version for some reason, so maybe this comment is not accurate anymore)